### PR TITLE
{CI} Test vm in serial mod for debian

### PR DESCRIPTION
**Pull request by AllyW**
**Related command**

**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

https://github.com/Azure/azure-cli/issues/28643

### CODE REVIEW

**Comment by yonzhan** [comment]
CI

File: scripts/release/debian/test_deb_package.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 pytest_base_cmd = '/opt/az/bin/python3 -m pytest -v --forked -p no:warnings --log-level=WARN'
 pytest_parallel_cmd = '{} -n auto'.format(pytest_base_cmd)
-serial_test_modules = ['botservice', 'network', 'cloud', 'appservice', 'iot', 'resource']
+serial_test_modules = ['botservice', 'network', 'cloud', 'appservice', 'iot', 'resource', 'vm']
```
Comment (jiasli) [inline scripts/release/debian/test_deb_package.py:16]: `serial_test_modules` is only used to mark these modules' tests as "serial", but being skipped.

Comment (AllyW) [reply scripts/release/debian/test_deb_package.py:16]: Pr title updates. 
Concurrent calls in vm test causes this issue in debian

**Comment by bebound** [comment]
I believe you also need to change `test_rpm_package.py`.
By the way, if https://github.com/Azure/azure-cli/issues/28643#issuecomment-2031072950 is true, then every command in codegen should run in serial. I don't think this is a good solution.

**Pull request closed without merging**